.github/workflows/docker_old_ver.yml

```diff
@@ -0,0 +1,169 @@
+name: docker_old_ver
+
+on:
+  push:
+    branches: [ main, firelight-test-march-2025 ]
+  pull_request:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+  packages: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref_name }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: metanorma/metanorma:1.12.3
```

Comment:
I suspect this won’t work, as Metanorma Docker images are not tagged per version. Discussing here: https://github.com/metanorma/metanorma-docker/issues/227